Colours.txt

```diff
@@ -27,14 +28,10 @@
 #define river 4915200
 #define road 6908265
 #define runway 13491405
+#define standHold 16777215
```

Comment:
Is this used for anything other than SMRs? Colours like this, terminal, apron, building, grass _sound_ like SMRy colours?? I think colours like road are used on both frrom memory, so that would be find, but I wonder if we should rename those that are purely SMR colours "smr____" e.g. `smrStandHold` if so, for consistency? Thoughts welcome.

Reply:
Grass was a difficult one as there are already 2 smr grass (1&2). I'll have a look at the rest.

Reply:
This is what I have come up with for colours that are purely used on smr Displays. Also fixes the capitalisation issues
```
    apron -> smrApron2
    arrester -> smrArrester
    building -> smrBuilding
    Edge -> smrEdge
    grass -> smrGrass3
    grey -> smrGrey2
    runway -> smrRunway
    standHold -> smrStandHold
    taxiway -> smrTaxiway
    terminal -> smrTerminal
    water -> smrWater
    Black -> black
    Blue -> smrBlue
```